.eslintrc

```diff
@@ -42,7 +42,7 @@
     "no-spaced-func": 2,
     "no-trailing-spaces": 2,
     "no-undef-init": 2,
-    "no-unused-expressions": 2,
+    "no-unused-expressions": 0,
```

Comment:
Don't need to turn off this rule for all code. Just add comments where it's necessary.

Reply:
I had to update `@typescript-eslint/parser` to use _nullish coalescing operator (??)_ here:
https://github.com/DevExpress/testcafe/pull/6541/files#diff-09938ba16ab6bb601c1e825c8106cac7f5c1435fbfd39bf231831fbb2ea5355cR26-R28

And after update this rule, it throws an error each time when finding the following code `this.compilerService?.on(...)`.

I'll fix it through the `if` condition.